### DISCUSSION TITLE
HELP! how to clear SQLite

### DISCUSSION THREAD

**Discussion by carlyyculver** [pawelsalawa/sqlitestudio#4832] [General]
Hi Everyone! I need some help, I tried pasting a query in my SQL lite a now its trying to process but it was way to large and is stuck now loading. Any one know how to force delete?? Ive tried force quitting, and uninstalling/ reinstalling SQLite studio but it wont clear! Please Help!!

**Comment by pawelsalawa** [comment]
Close SQLiteStudio. Delete config file (https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location). Start SQLiteStudio.

**Reply by lrobinsonco** [reply]
I have the same issue. No matter what i do to uninstall and reinstall the new application attempts to open the old file. I cannot locate the config file in my Library(Mac)

**Reply by pawelsalawa** [reply]
Run Terminal and there type:
```
ls -l $HOME/Library/Preferences/SalSoft/SQLiteStudio
```
What does it print out?
